kubernetes/istio-rbac/istio-rbac.yaml

```diff
@@ -0,0 +1,21 @@
+# Permissions and roles for istio
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: istio-manager-service-account
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: istio-manager-admin-role-binding
+subjects:
+- kind: ServiceAccount
+  name: istio-manager-service-account
+  namespace: default
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
I'm worrying that we are giving it cluster-admin role. This is against the principal of least privilege. We should find out a minimum set of permissions required to run `manager`.